.github/workflows/ci.yaml

```diff
@@ -26,6 +26,11 @@ jobs:
           role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
           aws-region: eu-west-1
 
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.13.0'
```

Comment:
Could we make this [`node-version-file: .nvmrc`](https://github.com/actions/setup-node/blob/869f4dd0c7f320ae834c2724d92a364de3893c24/action.yml#L10-L11) instead? The benefit being as the `.nvmrc` file changes, CI will keep in sync.

Reply:
👍